opencensus/exporters/stats/stackdriver/internal/stackdriver_utils.cc

```diff
@@ -153,15 +149,28 @@ std::vector<google::monitoring::v3::TimeSeries> DataToTimeSeries(
 
 }  // namespace
 
+std::string MakeType(absl::string_view metric_name_prefix,
+                     absl::string_view view_name) {
+  return absl::StrCat(metric_name_prefix, view_name);
+}
+
+bool IsKnownCustomMetric(absl::string_view metric_type) {
+  return absl::StartsWith(metric_type, "custom.googleapis.com/") ||
+         absl::StartsWith(metric_type, "external.googleapis.com/");
+}
+
 void SetMetricDescriptor(
     absl::string_view project_name, absl::string_view metric_name_prefix,
     const opencensus::stats::ViewDescriptor& view_descriptor,
     google::api::MetricDescriptor* metric_descriptor) {
   metric_descriptor->set_name(
       MakeName(project_name, metric_name_prefix, view_descriptor.name()));
-  metric_descriptor->set_type(
-      MakeType(metric_name_prefix, view_descriptor.name()));
-  SetOpenCensusTaskLabelDescriptor(metric_descriptor->add_labels());
+  const std::string metric_type =
+      MakeType(metric_name_prefix, view_descriptor.name());
+  metric_descriptor->set_type(metric_type);
+  if (IsKnownCustomMetric(metric_type)) {
```

Comment:
Probably add a comment to say why.

Reply:
https://github.com/census-instrumentation/opencensus-cpp/pull/385